tests/conftest.py

```diff
@@ -138,3 +138,17 @@ def pytest_configure(config):
             enabled_remotes.discard(remote_name)
         if enabled:
             enabled_remotes.add(remote_name)
+
+
+@pytest.fixture
+def dummy_stage(tmp_dir, dvc):
```

Comment:
If the idea sticks, I can refactor tests using `create_stage`, there is quite few of them.